batchflow/models/tf/encoder_decoder.py

```diff
@@ -111,17 +112,21 @@ class EncoderDecoder(TFModel):
     @classmethod
     def default_config(cls):
         config = TFModel.default_config()
+
         config['body/encoder'] = dict(base=None, num_stages=None)
         config['body/encoder/downsample'] = dict(layout='p', pool_size=2, pool_strides=2)
         config['body/encoder/blocks'] = dict(base=cls.block)
-        config['body/embedding'] = dict(base=cls.block)
+
+        config['body/embedding'] = dict()
+        config['body/embeddings/common'] = dict(base=cls.block)
```

Comment:
Having both `embedding` and `embeddings` is very confusing. Let's allow `embedding` to be a list of dicts when configuring multiple embeddings